k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -1,98 +1,123 @@
-#!/bin/bash

```

Comment:
Git diff is showing the whole file as changed since each line of the script had `\r` char at the end, which was resulting in the run time error on centos 7.9.
Removing the character solved the issue. but now the complete file is shown as changed. should I take back that change?

Reply:
So dos2unix is still not applied to this file before checkin? 

If you have addressed that, i think its fine, such change will show whole file change only. Just mention the line numbers which are changed, so we can review properly

Reply:
yes.. marked the line numbers 👍

Reply:
where @sumedhak27

Reply:
with recent changes, the `\r`  char was removed so now git diff is showing my changes correctly.